src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -362,9 +362,9 @@ internal static bool HasParameterList(CrefSyntax crefSyntax)
             return false;
         }
 
-        private static SymbolInfo GetCrefSymbolInfo(ImmutableArray<Symbol> symbols, SymbolInfoOptions options, bool hasParameterList)
+        private static SymbolInfo GetCrefSymbolInfo(OneOrMany<Symbol> symbols, SymbolInfoOptions options, bool hasParameterList)
         {
-            switch (symbols.Length)
+            switch (symbols.Count)
```

Comment:
note: i'd be totally fine with adding a Length property to keep this diff smaller.  it would be redundant with .Count.  Up to you guys.